tic-tac-toe.js

```diff
@@ -0,0 +1,13 @@
+const x = 1
```

Comment:
Remember to end statements with semicolons.

Reply:
done